### ISSUE TITLE
DeleteCut on iOS 16

### ISSUE THREAD

**Issue by Invisible40** [Loy6410/ios16-tweaks#43]
I do not think DeleteCut works on iOS 16. The tweak itself is not rootless, and I have tried derootifier, only to be met with my phone crashing whenever I open an app.

Using iOS 16.0.1 and Dopamine

**Comment by Loy6410** [comment]
> I do not think DeleteCut works on iOS 16. The tweak itself is not rootless, and I have tried derootifier, only to be met with my phone crashing whenever I open an app.
> 
> 
> 
> Using iOS 16.0.1 and Dopamine

I've just tried to install it and after converting successfully with derootifier I installed the tweak and did a respring. The tweak works perfectly for me, if you got any errors in the derootifier after converting you should install those tweaks from sileo:
- odcctools
- file
- ldid
- fakeroot

I'm working on updating this tweak list for dopamine btw, so I'll include the rootless converted deb.

**Comment by Invisible40** [comment]
Yeah unfortunately doing the derootifier causes my apps to crash, and when I go to respring, it causes my phone to not start up (so I have to force restart to get rid of the tweak). Let me know when you update the thread so I can try your rootless deb.
